spec/ruby/optional/capi/string_spec.rb

```diff
@@ -784,4 +784,16 @@ def to_str
       @s.rb_String({"bar" => "foo"}).should == '{"bar"=>"foo"}'
     end
   end
+
+  describe "SafeStringValue" do
+    it "raises for tained string when $SAFE is 1" do
+      t = Thread.new {
+        $SAFE = 1
+        lambda {
+          @s.SafeStringValue("str".taint)
+        }.should raise_error(SecurityError)
+      }
+      t.join
+    end
```

Comment:
I think it would be interesting to test the cases where it does not raise, similar to the specs of StringValue above.

Reply:
I did it (d9d40d2).

Reply:
@eregon Thanks!